.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "matrix-doc"]
```

Comment:
It makes it fairly easy for someone developing to run tests but quite difficult if you've just installed a release version and want to run the tests.